templates/cisco_ios_show_environment_temperature.template

```diff
@@ -0,0 +1,41 @@
+Value SWITCH (\d+)
+Value SWITCH_TEMPERATURE (\w+)
+Value INLET_TEMPERATURE_VALUE (\d+)
+Value INLET_TEMPERATURE_STATE (\w+)
```

Comment:
```suggestion
Value INLET_TEMPERATURE_STATE (\S+)
```

Reply:
tested and works.  Moving to next suggestion